src/actions/send.js

```diff
@@ -0,0 +1,40 @@
+import { createActions } from 'redux-actions';
+
+import { showAlert } from '../utils/alerts';
+import { wallet } from '../utils/wallet';
+
+export const transfer = ({ contractName, amount, memo, receiverId, isStorageBalanceAvailable }) => async (dispatch, gesState) => {
+    if (contractName && !isStorageBalanceAvailable) {
+        await dispatch(send.transfer.storageDeposit(contractName, receiverId));
+    }
+
+    const { transaction, status } = await dispatch(send.transfer[contractName ? 'TOKENS' : 'NEAR'](contractName, amount, memo, receiverId));
+
+    if (status?.SuccessValue) {
```

Comment:
It seems like this will silently fail if there is an error? What should be happening here if there is an error transferring or the `successValue` isn't set in the response? Feels like we're missing some actions and wrapping code here

Reply:
If tx fails, an error will be thrown and the user will see Global Alert. After that, the user will see this tx in the Recent activity box, with `failed` indicator.

From the UI site, I think we should show a screen with information that the tx fails, and link to receipt like on success.